sources/packages/api/src/route-controllers/student/models/student.dto.ts

```diff
@@ -91,15 +91,22 @@ export interface StudentFileUploaderForm {
 /**
  *  Student uploader interface
  */
-export interface StudentFileUploaderDto {
+export class StudentFileUploaderDTO {
```

Comment:
This is a payload for patch. so class validators must be used here.

Reply:
I also believe that without class validators, when using a class API will not be able to parse the properties.